src/main.cpp

```diff
@@ -84,6 +93,8 @@ int main()
 	initialize_pins();
 	int frequency = 1000;
 
+	gpio_put(pinU_L, 1);
```

Comment:
Issue: technically superfluous since this function will soon be rewritten, but only the test circuit needs to have the enable input set.

Reply:
Sorry, I forgot to remove this line from a previous commit, since `set_logic_pin_` does it anyways.